data_pipeline_api/java_api/src/test/java/uk/ramp/file/FileDirectoryNormaliserTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.ramp.file;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileDirectoryNormaliserTest {
+  private String parentPath;
+
+  @Before
+  public void setUp() throws Exception {
+    parentPath = "parentPath/";
+  }
+
+  @Test
+  public void normaliseRelativePath() {
```

Comment:
Are the names of these two tests the wrong way round?  This test is normalising an absolute path, and the following test is normalising a relative path.

Reply:
You're correct. I've updated now. Thanks.